### PR TITLE
Update the installation

### PR DESCRIPTION
**Pull request by yuexiao-wang**
Signed-off-by: yuexiao-wang <wang.yuexiao@zte.com.cn>

Add some details to update the installation

### CODE REVIEW

**Comment by yuexiao-wang** [comment]
@arschles PTAL

File: docs/install-1.7.md [hunk 1]
```diff
@@ -55,8 +53,14 @@ your `minikube start` command with this flag:
...
 ```
+If you are using `hack/local-up-cluster.sh`, ensure the
+`AUTHORIZATION_MODE` environment variable is set as follows:
+
+``console
+AUTHORIZATION_MODE=Node,RBAC hack/local-up-cluster.sh -O
+```
 
-AssumingBy default, `helm init` installs the Tiller pod into the `kube-system`
+Assuming by default, `helm init` installs the Tiller pod into the `kube-system`
```
Comment (MHBauer) [inline docs/install-1.7.md]: I don't think we need the word "assuming" here.

Comment (yuexiao-wang) [reply docs/install-1.7.md]: Yes, I agree it. At first, I deal with it as a typos

**Comment by yuexiao-wang** [comment]
@MHBauer Thanks for your review and it is done.

File: docs/install-1.7.md [hunk 2]
```diff
@@ -18,8 +18,6 @@ authentication and authorization.
 
 The Service Catalog utilizes API aggregation to present its API.
 
-# Prerequisites
```
Comment (MHBauer) [inline docs/install-1.7.md]: We should either keep this one and get rid of the next line, or adjust all the step numbering. Right now this doc has "Step 1" twice.

Arguably, given the size of the prereqs,we should do the renumbering.

File: docs/install-1.7.md [hunk 3]
```diff
@@ -125,11 +129,11 @@ Finally, create the certs:
 source ../../contrib/svc-cat-apiserver-aggregation-tls-setup.sh
 ```
 
-# Step 2 - Install the Helm Chart
+# Step 2 - Install Service Catalog with Helm Chart
 
 Use helm to install the Service Catalog, associating it with the
-configured name ${HELM_NAME}, and into the specified namespace." This
-command also enables authentication and aggregation and provides the
+configured name ${HELM_RELEASE_NAME}, and into the specified namespace." This
```
Comment (MHBauer) [inline docs/install-1.7.md]: good catch.

**Comment by MHBauer** [comment]
@yuexiao-wang , one more change please. Sorry I did not catch it the first time.

File: docs/install-1.7.md (hunk 2 above)
Comment (kibbles-n-bytes) [reply docs/install-1.7.md]: I agree with doing the renumbering.

Comment (yuexiao-wang) [reply docs/install-1.7.md]: I am OK to do the renumbering

**Comment by yuexiao-wang** [comment]
@MHBauer This PR is updated. Any other suggestions?

**Comment by yuexiao-wang** [comment]
@pmorie @MHBauer I am paying close attention to service-catalog and interest in contributing. If have some tasks in service-catalog, please assgin me to complete it. 
Thanks

**Pull request merged**